### PR TITLE
MODAUD-31 - Requests log actions - improvements

### PR DESCRIPTION
**Pull request by khandramai**
[MODAUD-31](https://issues.folio.org/browse/MODAUD-31) - Requests log actions - improvements

## Purpose
Improvements for request log entries

## Approach
Mapping improvement

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-audit-server/src/main/java/org/folio/builder/service/RequestRecordBuilder.java [hunk 1]
```diff
@@ -1,49 +1,34 @@
...
-import static org.folio.util.LogEventPayloadField.PAYLOAD;
-import static org.folio.util.LogEventPayloadField.REORDERED;
-import static org.folio.util.LogEventPayloadField.REQUESTER;
-import static org.folio.util.LogEventPayloadField.REQUESTS;
-import static org.folio.util.LogEventPayloadField.REQUEST_ID;
-import static org.folio.util.LogEventPayloadField.SERVICE_POINT_ID;
-import static org.folio.util.LogEventPayloadField.STATUS;
-import static org.folio.util.LogEventPayloadField.UPDATED;
-import static org.folio.util.LogEventPayloadField.USER_ID;
+import static org.folio.util.LogEventPayloadField.*;
```
Comment (siarhei-charniak) [inline mod-audit-server/src/main/java/org/folio/builder/service/RequestRecordBuilder.java:12]: Please refactor according to code style

File: mod-audit-server/src/main/java/org/folio/builder/service/RequestRecordBuilder.java [hunk 2]
```diff
@@ -1,49 +1,34 @@
 package org.folio.builder.service;
 
-import static org.folio.builder.LogRecordBuilderResolver.REQUEST_CANCELLED;
-import static org.folio.builder.LogRecordBuilderResolver.REQUEST_CREATED;
-import static org.folio.builder.LogRecordBuilderResolver.REQUEST_MOVED;
-import static org.folio.builder.LogRecordBuilderResolver.REQUEST_REORDERED;
-import static org.folio.builder.LogRecordBuilderResolver.REQUEST_UPDATED;
+import static java.util.stream.Collectors.toMap;
+import static org.folio.builder.LogRecordBuilderResolver.*;
```
Comment (siarhei-charniak) [inline mod-audit-server/src/main/java/org/folio/builder/service/RequestRecordBuilder.java:4]: Please refactor according to code style

File: mod-audit-server/src/main/java/org/folio/builder/service/LogRecordBuilder.java [hunk 3]
```diff
@@ -15,26 +16,11 @@
...
-import static org.folio.util.LogEventPayloadField.ITEM_ID;
-import static org.folio.util.LogEventPayloadField.LAST_NAME;
-import static org.folio.util.LogEventPayloadField.NAME;
-import static org.folio.util.LogEventPayloadField.PERSONAL;
-import static org.folio.util.LogEventPayloadField.PERSONAL_NAME;
-import static org.folio.util.LogEventPayloadField.TEMPLATE_ID;
-import static org.folio.util.LogEventPayloadField.TEMPLATE_NAME;
-import static org.folio.util.LogEventPayloadField.USER_BARCODE;
-import static org.folio.util.LogEventPayloadField.USER_ID;
+import static org.folio.util.LogEventPayloadField.*;
```
Comment (siarhei-charniak) [inline mod-audit-server/src/main/java/org/folio/builder/service/LogRecordBuilder.java:19]: Please refactor according to code style

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -117,7 +117,8 @@
             "users.collection.get",
             "templates.item.get",
             "inventory.items.item.get",
-            "inventory-storage.holdings.item.get"
+            "inventory-storage.holdings.item.get",
+            "circulation-storage.cancellation-reasons.collection.get"
```
[the remark this answers is not in the document]
Comment (khandramai) [reply descriptors/ModuleDescriptor-template.json:121]: Good catch! Thank you!

**Comment by khandramai** [comment]
Verified locally
Screen Shot 2020-11-11 at 11 09 30 AM

File: mod-audit-server/src/main/java/org/folio/builder/service/LogRecordBuilder.java [hunk 5]
```diff
@@ -142,7 +145,7 @@ public LogRecordBuilder(Map<String, String> okapiHeaders, Context vertxContext)
           JsonObject personal = getObjectProperty(userJson, PERSONAL);
           if (nonNull(personal)) {
```
Comment (siarhei-charniak) [inline mod-audit-server/src/main/java/org/folio/builder/service/LogRecordBuilder.java:146]:
```suggestion
          if (nonNull(personal) && nonNull(buildPersonalName(getProperty(personal, FIRST_NAME), getProperty(personal, LAST_NAME)))) {
```

**Review by siarhei-charniak — CHANGES_REQUESTED** [review]

**Review by siarhei-charniak — APPROVED** [review]

**Pull request merged**